web/src/main/resources/applicationContext-url-shortener.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@ersinciftci So does this apply to all properties now? That they can be overridden by system props? I guess except for anything defined in `GlobalProperties`

Reply:
@inodb it was always like this for all properties, that's how you could use `-Ddb.host` kind of things.

Reply:
@ersinciftci well `-Ddb.host` actually still doesn't work, or at least some of the `db.` parameters don't update when passed to jar/war through `-D`. Not sure why, maybe cause in some places we use `GlobalProperties` instead of the spring prop directly through annotation.

Why do we need a separate `applicationContext` for the shortener?

Seems like only this line `<context:component-scan base-package="org.cbioportal.url_shortener" />` is new compared to applicationContext.xml